content/en/docs/v3.6/tasks/operator/how-to-migrate.md

```diff
@@ -4,10 +4,73 @@ description: etcd v2 to v3 migration guide
 weight: 1200
 ---
 
-`migrate` to transform etcd v2 to v3 data:
+`migrate` to transform etcd v2 data model to v3 data model.
```

Comment:
```suggestion
`migrate` to transform etcd v2 data format to v3 data format.
```

"model" generally refers to the schema, rather than the storage format